LICENSE

```diff
@@ -0,0 +1,672 @@
+MIT License
+
+Copyright (c) 2071 Sam Wray
```

Comment:
:D

Reply:
lol, that was copied out of vcync/modV 😅